Revise footer and styling fixes

coreycaitlin opened the pull request:
Fixes issue(s) #1940 and #2064.

[![CircleCI](https://circleci.com/gh/18F/doi-extractives-data/tree/footer-fix.svg?style=svg)](https://circleci.com/gh/18F/doi-extractives-data/tree/footer-fix)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/footer-fix/)

Changes proposed in this pull request:

- remove sitemap from footer
- fix styling bug on social link (with an assist from @meiqimichelle)
- fix disbursements table styling

/cc @meiqimichelle @ericronne

ericronne commented:
Footer 👍 

Table could use a smidge of a tweak, if poss ...

**A** Left align headers
**B** Increase padding above and below content, so the rules aren't so close to it (i bumped from 5 to 10 here)

(Megachart is a good reference here)

![image](https://cloud.githubusercontent.com/assets/9259185/20401866/1ad068de-acc0-11e6-8582-9df565c05aec.png)

thx!

gemfarmer commented on _sass/elements/_tables.scss:
```diff
@@ -54,6 +54,7 @@ td {
 
   thead {
     background-color: transparent;
+    line-height: 1.2;
```
This could be `line-height: $light-height-caps;`

It will compute to `line-height: 1.2rem;`

coreycaitlin commented:
- Fixed top/bottom padding in the table
- Left-aligned headers
- Moved the `$line-height-caps` variable to the variables file so it's easier to find next time! Also, this made it stop failing to build.

Hopefully good to go now!

gemfarmer commented:
LGTM!

ericronne commented:
👍
